olog.py

```diff
@@ -1,12 +1,9 @@
-from datetime import datetime
-import requests
-import requests.auth
+from httpx import AsyncClient
+import asyncio
 
 
 __all__ = ['Client']
 
-headers = {'content-type': 'application/json', 'accept': 'application/json'}
```

Comment:
Why not specify these?

Reply:
`json` parameter like [here](https://github.com/NSLS-II/pyolog2/pull/10/files#diff-4b602154de7ca3e54b513607859e7ff2R43) is used when post/put. It will update content-type to `application/json`.
`res.json()` basiclly declare client only _accept_  json.

Reply:
When I tried to experiment with changing the headers, it looked like the standard naming is to use capitalized keys, such as `Content-Type`.

Reply:
Sounds good.